doc/audio.md

```diff
@@ -13,75 +22,75 @@ flutter:
+use of optimized features that allow sounds to be looped without gaps between their iterations, and
+almost no drop on the game frame rate will happen. You should whenever possible, prefer these
+methods.
 
-The difference is that each instance shares a different cache. Normally you would want to use the `Flame.audio` instance and totally share the cache.
-
-You must have an appropriate folder structure and add the files to the `pubspec.yaml` file, as explained above.
-
-It can be an MP3, OGG or a WAV file (those are the ones I tested and it worked)
+`playLongAudio/loopLongAudio` allows for audios of any length to be played, but they do create frame
+rate drop, and the looped audio will have a small gap between iterations.
 
-This uses the [audioplayers](https://github.com/luanpotter/audioplayer) lib, in order to allow playing multiple sounds simultaneously (crucial in a game). You can check the link for more in-depth explanations.
+You can use [the `Bgm` class](bgm.md) (via `FlameAudio.bgm`) to play looping background music
+tracks. The `Bgm` class lets Flame manage the auto pausing and resuming of background music tracks
+when pausing/resuming the game.
 
-If you want to play indefinitely, just use `loop` function:
+The file formats that have been tested and works are MP3, OGG and WAV.
 
-```dart
-Flame.audio.loop('music.mp3'); // Or
-Flame.audio.loopLongAudio('music.mp3');
-```
-
-Alternatively, you can use [the `Bgm` class (via `Flame.bgm`)](bgm.md) to play looping background music tracks. The `Bgm` class lets Flame manage the auto pausing and resuming of background music tracks when pausing/resuming the game/app.
-
-The difference between the `play/loop` and `playLongAudio/loopLongAudio` is that `play/loop` makes uses of optimized features that allow sounds to be looped without gaps between their iterations, and almost no drop on the game frame rate will happen. You should whenever possible, prefer these methods. `playLongAudio/loopLongAudio` allows for audios of any length to be played, but they do create frame rate drop, and the looped audio will feature a small gap between iterations.
+This bridge library (flame_audio) uses [audioplayers](https://github.com/luanpotter/audioplayer) in
+order to allow for playing multiple sounds simultaneously (crucial in a game). You can check the
+link for a more in-depth explanation.
 
-Finally, you can pre-load your audios. Audios need to be stored in the memory the first time they are requested; therefore, the first time you play each mp3 you might get a delay. In order to pre-load your audios, just use:
+Finally, you can pre-load your audios. Audios need to be stored in the memory the first time they
+are requested; therefore, the first time you play each mp3 you might get a delay. In order to
+pre-load your audios, just use:
 
 ```dart
-Flame.audio.load('explosion.mp3');
+FlameAudio.audioCache.load('explosion.mp3');
```

Comment:
I would add the await keyword here just to hind to the reader that this is async

Reply:
and on the other isolated snippets as well